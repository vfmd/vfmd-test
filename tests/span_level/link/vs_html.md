Containing:

[link containing <s>html</s>](url)


Contained:

html <s>containing [link](url)</s>

html <s>containing [link][ref]</s>

[ref]: /url


Intertwined:

[link <s>intertwined](url) with html</s>

link <s>intertwined [with html</s>](url)

