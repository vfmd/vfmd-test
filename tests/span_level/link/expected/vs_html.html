<p>Containing:</p>

<p><a href="url">link containing <s>html</s></a></p>

<p>Contained:</p>

<p>html <s>containing <a href="url">link</a></s></p>

<p>html <s>containing <a href="/url">link</a></s></p>

<p>Intertwined:</p>

<p>[link <s>intertwined](url) with html</s></p>

<p>link <s>intertwined [with html</s>](url)</p>
